shimmingtoolbox/utils.py

```diff
@@ -14,7 +14,7 @@
 logger = logging.getLogger(__name__)
 
 HOME_DIR = str(Path.home())
-PATH_ST_VENV = f"{HOME_DIR}/shimming-toolbox/python/envs/st_venv/bin"
+PATH_ST_VENV = f"{HOME_DIR}/shimming-toolbox/python/bin"
```

Comment:
Huh... this shouldn't need to be here. We should be using `__file__` or [`importlib.resources.files().joinpath()`](https://importlib-resources.readthedocs.io/en/latest/using.html).

But that's something to fix in another PR.